src/components/Header/BtnAdd/BtnAdd.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import BtnCSS from './BtnAdd.module.css';
+
+const BtnAdd = ()=>(
+    <div className={BtnCSS.div_btnAdd}>
```

Comment:
you can simplify it to single element <div classname={.....} />

Reply:
I'll do it. Thanks